src/components/modules/dragNDrop.ts

```diff
@@ -36,14 +47,23 @@ export default class DragNDrop extends Module {
    * Add drag events listeners to editor zone
    */
   private enableModuleBindings(): void {
-    const { UI } = this.Editor;
+    const { UI, BlockManager } = this.Editor;
 
     this.readOnlyMutableListeners.on(UI.nodes.holder, 'drop', async (dropEvent: DragEvent) => {
       await this.processDrop(dropEvent);
     }, true);
 
-    this.readOnlyMutableListeners.on(UI.nodes.holder, 'dragstart', () => {
-      this.processDragStart();
+    this.readOnlyMutableListeners.on(UI.nodes.holder, 'dragstart', (startDragEvent: DragEvent) => {
+      this.processDragStart(startDragEvent);
+    });
```

Comment:
```suggestion
    this.readOnlyMutableListeners.on(UI.nodes.holder, 'dragstart', (dragStartEvent: DragEvent) => {
      this.processDragStart(dragStartEvent);
    });
```

Just to have the same code style :)